upup/pkg/fi/cloudup/bootstrapchannelbuilder/cilium.go

```diff
@@ -46,12 +43,27 @@ func addCiliumAddon(b *BootstrapChannelBuilder, addons *api.Addons) {
 					Id:       id,
 				})
 			}
-		} else {
+		} else if ver.Minor == 8 {
 			version := "1.8.0-kops.1"
 			{
 				id := "k8s-1.12"
 				location := key + "/" + id + "-v1.8.yaml"
 
+				addons.Spec.Addons = append(addons.Spec.Addons, &api.AddonSpec{
+					Name:               fi.String(key),
+					Version:            fi.String(version),
+					Selector:           networkingSelector(),
+					Manifest:           fi.String(location),
+					Id:                 id,
+					NeedsRollingUpdate: "all",
+				})
+			}
+		} else if ver.Minor == 9 {
```

Comment:
Previously there was a lone `} else {`, was this acting as a default cilium version if the api field isn't set? If so, we should probably keep the `} else {` here so that 1.9 is installed by default.

Reply:
The code expects that the code running prior does the right thing. Added some guards against that not being the case now.

Reply:
Was just thinking about that error handling.